doc/release-notes/0.35/0.35.md

```diff
@@ -49,45 +49,45 @@ The following table covers notable changes in v0.35.0. Further information about
 <tbody>
 
 <tr>
-<td valign="top"></td>
-<td valign="top"></td>
-<td valign="top"></td>
-<td valign="top"></td>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/15555">#15555</a></td>
+<td valign="top"> Java&trade; dump output contains a new <tt>Initializing thread: <thread_name></tt> string in the existing <tt>3XMTHREADBLOCK</tt> line.</td>
```

Comment:
`<thread_name>` isn't showing up in the preview, I expect the `<`, `>` characters need to be handled specially.

Reply:
Corrected it.